packages/rlc-common/src/helpers/importsUtil.ts

```diff
@@ -95,3 +96,61 @@ export function getImportSpecifier(
     ""
   );
 }
+
+export function addImportToSpecifier(
```

Comment:
add the addImportToSpecifier helper here. And also add helper functions about clear import set and addImportsToFiles. @xirzec